Core/source/mobile-interface/validate-input.ts

```diff
@@ -207,26 +203,20 @@ const hasProp = (
       && value.filter((x: any) => typeof x === 'string').length === value.length;
   }
   if (type === 'PrvKeyInfo?') {
-    if (value === null) {
+    if (value === null) { // eslint-disable-line no-null/no-null
```

Comment:
I saw many `// eslint-disable-line no-null/no-null` lines.
I think we use `null` a lot in this project and I don't think `null` is unsafe to use.
Maybe we can change eslint setting to disable `no-null/no-null` rule?